outscale/resource_outscale_access_key_test.go

```diff
@@ -162,15 +201,28 @@ func testAccCheckOutscaleAccessKeyDestroy(s *terraform.State) error {
 }
 
 const testAccOutscaleAccessKeyBasicConfig = `
-	resource "outscale_access_key" "outscale_access_key" {
-               expiration_date = "2025-01-04"
-        }`
+	resource "outscale_access_key" "outscale_access_key" {}`
 
 func testAccOutscaleAccessKeyUpdatedConfig(state string) string {
 	return fmt.Sprintf(`
 		resource "outscale_access_key" "outscale_access_key" {
```

Comment:
In case, we change the resource name, the test will be wrong, maybe we could parametrize resource name with `resourceName`